openstates/in/bills.py

```diff
@@ -77,9 +81,127 @@ def get_name(self,random_json):
+            text = convert_pdf(path, 'text')
+            lines = text.split("\n")
+            os.remove(path)
+
+
+            chamber = "lower" if "house of representatives" in lines[0].lower() else "upper"
+            date_parts = lines[1].strip().split()[-3:]
+            date_str = " ".join(date_parts).title() + " " + lines[2].strip()
+            vote_date = datetime.datetime.strptime(date_str,"%b %d, %Y %I:%M:%S %p")
+
+            for res,val in result_types.items():
+                if res in lines[3].upper():
+                    passed = val
+                    break
+
+            motion = " ".join(lines[4].split()[:-2])
+            try:
+                yeas = int(lines[4].split()[-1])
+                nays = int(lines[5].split()[-1])
+                excused = int(lines[6].split()[-1])
+                not_voting = int(lines[7].split()[-1])
+            except ValueError:
+                self.logger.warning("Vote format is weird, skipping")
+                continue
+            other_count = excused + not_voting
+
+            vote = Vote(chamber,vote_date,motion,passed,yeas,nays,other_count,yes_votes=[],no_votes=[],other_votes=[])
+
+            vote.add_source(proxy_link)
+
+            currently_counting = ""
+
+            possible_vote_lines = lines[8:]
+            for l in possible_vote_lines:
+                l = l.replace("\xc2\xa0"," -")
+                if "yea -" in l.lower():
+                    currently_counting = "yes_votes"
+                elif "nay -" in l.lower():
+                    currently_counting = "no_votes"
+                elif "excused -" in l.lower():
+                    currently_counting = "other_votes"
+                elif "not voting -" in l.lower():
+                    currently_counting = "other_votes"
+                elif currently_counting == "":
+                    pass
+                elif re.search(r'v\. \d\.\d',l):
+                    #this gets rid of the version number
+                    #which is often found at the bottom of the doc
+                    pass
+                else:
+                    voters = l.split("  ")
+                    for v in voters:
+                        if v.strip():
+                            vote[currently_counting].append(v.strip())
+
+            assert (len(vote["yes_votes"]) == vote["yes_count"],
+                "Yes vote counts ({count}) don't match count of actual votes ({actual})".format(count=vote["yes_count"],actual=len(vote["yes_votes"])))
+            assert (len(vote["no_votes"]) == vote["no_count"],
+                "No vote counts ({count}) don't match count of actual votes ({actual})".format(count=vote["no_count"],actual=len(vote["no_votes"])))
+            assert (len(vote["other_votes"]) == vote["other_count"],
+                "Other vote counts ({count}) don't match count of actual votes ({actual})".format(count=vote["other_count"],actual=len(vote["other_votes"])))
```

Comment:
Same issue that the Prof noted in my recent pull request:
Even `assert (False, "message")` will pass, because Python assumes you're checking for an empty tuple, not that you're just doing a line continuation.